README.adoc

```diff
@@ -83,14 +83,15 @@ is in the `start/src` directory:
 
 If you do not have Gradle installed, make sure that the `JAVA_HOME` environment variable is set, or that the Java application can run. Running the Gradle Wrapper automatically installs Gradle. To learn more about the Gradle Wrapper, see the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Gradle Wrapper documentation^].
 
-Ensure that you are in the `start` directory. Test that Gradle is installed properly by running the following command:
+Test that Gradle is installed properly by navigating to the `start` directory and running the following command:
```

Comment:
I think it is awkward that we ask the user to navigate to the `start` directory and then have them run a set of commands that include `cd start`. I think it should only be one or the other.

Reply:
On second thought, I think this can be resolved by wording it like the following:

> Navigate to the `start` directory and verify Gradle's installation. 

Leaving the code block below this paragraph as the same.

Reply:
I replaced it with the following:

> Run the following commands to navigate to the `start` directory and verify that Gradle has been installed correctly:

What do you think? I wanted to make sure I preserved the instruction to "run the commands" so that its clear what the user is being asked to do.